packages/bsky/tests/views/__snapshots__/notifications.test.ts.snap

```diff
@@ -245,24 +245,7 @@ Array [
     "cid": "cids(12)",
     "indexedAt": "1970-01-01T00:00:00.000Z",
     "isRead": false,
-    "labels": Array [
-      Object {
-        "cid": "cids(12)",
-        "cts": "1970-01-01T00:00:00.000Z",
-        "neg": false,
-        "src": "did:example:labeler",
-        "uri": "record(14)",
-        "val": "test-label",
-      },
-      Object {
-        "cid": "cids(12)",
-        "cts": "1970-01-01T00:00:00.000Z",
-        "neg": false,
-        "src": "did:example:labeler",
-        "uri": "record(14)",
-        "val": "test-label-2",
-      },
-    ],
+    "labels": Array [],
```

Comment:
hmmm these were good integration tests that labels were being hydrated correctly - think we could just add these labels directly in the seed scripts?

not a _huge_ deal, but we don't currently have separate tests for label hydration 😬

Reply:
Seems like a good idea!  Will take a look at that.